cmake-docker-environment/labs/cmake-example/readme.md

```diff
@@ -0,0 +1,18 @@
+# cmake-example
+These are similar to the make-example exercises. You'll see how a single  CMakeLists.txt file generates a MakeFile file for a variety of system configurations.
+
+## Exercise 1: Generate the MakeFile file and build the executable
+   ```
+   $ cd cmake-example
+   $ mkdir build && cd build
+   $ cmake ..
+   $ make
+   $ ./greeting
+   ```
+
+## Exercise 2: Examine the flexibility of CMake's MakeFile file
+Make a slight change to greeting.cpp and build again.
+   ```
+   $ make
+   $ ./greetingTest
```

Comment:
why the executable name has changed? It seems the exercise is only changing the c++ code, not the CMakeLists.txt

Reply:
I've fixed line 17 in a later commit to say ./greeting. Correct, the goal of this exercise is to show how the make file created by cmake offers the same time-saving benefits as the hand-written make file by recompiling only files that have changed. Later exercises will ask students to modify an existing CMakeLists.txt or write a CMakeLists.txt file from scratch.